src/SFA.DAS.EmployerIncentives.Domain/Accounts/LegalEntity.cs

```diff
@@ -38,5 +55,25 @@ public void SignedAgreement(int signedAgreementVersion, int minimumRequiredVersi
                 Model.HasSignedAgreementTerms = true;
             }
         }
+
+        public void SetHashedLegalEntityId(string hashedId)
+        {
+            Model.HashedLegalEntityId = hashedId;
+        }
+
+        internal void UpdateVendorRegistrationCaseStatus(string caseId, string vendorId, string status, DateTime caseStatusLastUpdatedDate)
+        {
+            if (VrfStatusIsCompleted()) return;
```

Comment:
This logic looks wrong to me. I understood that we are supposed to ignore the incoming update if the new status is "Case request complete" . This logic updates the record with the status "Case request complete" and the all subsequent updates are then ignored.

Reply:
The logic is correct, it should set it to complete once and then ignore all future updates.

Reply:
Yes, however we agreed to remove VendorID from the status update. My bad i shouldn't have added it in the first place.